stdlib/float.ml

```diff
@@ -73,10 +84,20 @@ external sinh : float -> float = "caml_sinh_float" "sinh"
   [@@unboxed] [@@noalloc]
 external tanh : float -> float = "caml_tanh_float" "tanh"
   [@@unboxed] [@@noalloc]
+external trunc : float -> float = "caml_trunc_float" "caml_trunc"
+  [@@unboxed] [@@noalloc]
+external round : float -> float = "caml_round_float" "caml_round"
+  [@@unboxed] [@@noalloc]
 external ceil : float -> float = "caml_ceil_float" "ceil"
   [@@unboxed] [@@noalloc]
 external floor : float -> float = "caml_floor_float" "floor"
 [@@unboxed] [@@noalloc]
+
+let is_integer x = x = trunc x && is_finite x
+
+external nextafter : float -> float -> float
+  = "caml_nextafter_float" "caml_nextafter" [@@unboxed] [@@noalloc]
```

Comment:
How does this primitive behaves on error conditions (cf http://pubs.opengroup.org/onlinepubs/009695399/functions/nextafter.html)?

Reply:
They just wrap the function you link to (the reason for the `caml_nextafter` is to be able to use a macro to work around the old windows `_nextafter`).

Reply:
I think it is worth mentioning the error behavior.  Does it return nan, does it raise an exception?

Reply:
I added some explanation on the last commit below.  OK with you?